.github/workflows/generate-changelog/action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
     required: true
   BASE_TAG_NAME:
     required: true
+  GENERATE_EMPTY_CHANGELOG:
+    required: false
+    default: false
```

Comment:
Don't we always want to generate an empty change log if no results?

Reply:
Yes, but this action is already in use today in publish cloud, which we don't upload the changelog to the artifact. (Different behavior) So I don't want to impact on existing actions that use this composite. The empty changelog is used only for the new `create-rc-tag` action.